src/pages/index.astro

```diff
@@ -2,64 +2,82 @@
 import BaseHead from '../components/BaseHead.astro';
 import Header from '../components/Header.astro';
 import Footer from '../components/Footer.astro';
-import { SITE_TITLE, SITE_DESCRIPTION } from '../consts';
+import { SITE_TITLE, SITE_DESCRIPTION} from '../consts';
+import {background_config} from '../consts'
+import Particles from "astro-particles";
+import type { ISourceOptions } from "tsparticles-engine";
+
+const options: ISourceOptions = background_config
 ---
 
 <!doctype html>
 <html lang="en">
-	<head>
-		<BaseHead title={SITE_TITLE} description={SITE_DESCRIPTION} />
-		<script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r128/three.min.js" defer></script>
+    <head>
+        <script type="module">FDE49E
+            import astroParticles from 'https://cdn.jsdelivr.net/npm/astro-particles@2.10.0/+esm'
+        </script>
+        <BaseHead title={SITE_TITLE} description={SITE_DESCRIPTION} />
         <script src="/scripts/globe.js" defer></script>
```

Comment:
Looks like we're still using globe.js? This line + that file can be deleted.

Reply:
I will see all the suggestion and make a commit soon :)
Thank you